index.ts

```diff
@@ -8,91 +8,43 @@ import {
 import env from "./env.json";
 import MyClient from "./MyClient";
 
-type NestedToken = Omit<TokenByIdResult, "nestingChildTokens"> & {
-  nestingChildTokens: NestedToken[];
-};
+(async function main() {
+  console.log("Started", env);
+  await MyClient.init();
+  console.log("connected");
+  console.time("magic1");
 
-const MAX_BUNDLE_DEPTH = 5;
 
-async function getTopmostParent(
-  this: Sdk,
-  collectionId: number,
-  tokenId: number
-): Promise<NestedToken> {
-  const parent = await this.tokens.parent({ collectionId, tokenId });
-  if (parent)
-    return getTopmostParent.call(this, parent.collectionId, parent.tokenId);
-  const token = await this.tokens.get_new({ collectionId, tokenId });
-  if (!token) throw new Error("Token was not found");
-  return { ...token, nestingChildTokens: [] };
-}
+  
+  for(let i = 0; i < 1; i++) { // update 'i < 1' accordingly if you need to get more than 1 token
```

Comment:
more than one collection, pehaps?

In that case it might be better to put into config(env)

Reply:
done